src/components/Pages/Admin/Designer.vue

```diff
@@ -362,6 +367,48 @@ export default {
             );
         },
 
+        // Send selected toolbar and perspectives to first student in selected group/ groups.
+        async sendToFirstInGroup(groups) {
+            if (!groups.length) {
+                this.showToast('Please select group to send toolbar for first student', 'warning');
+            } else {
+                const information = this.prepareMetaInformationPerspectivesAndToolbars();
+
+                await this.findGroupsInStore({
+                    query: {
+                        name: groups,
+                        class: this.code,
+                    },
+                });
+
+                const groupsObjects = await this.findGroups({
+                    query: {
+                        name: groups,
+                        class: this.code,
+                    },
+                });
+
+                const properties = {
+                    perspectives: information.perspectives,
+                    toolbar: information.toolbar,
+                };
+
+                for (let i = 0; i < groupsObjects.length; i += 1) {
+                    const g = groupsObjects[i];
```

Comment:
Also, prefer destructuring to the `.` accessors:

```javascript
const { _id } = groupsObjects[i];
```

If you change the `for` loop to the `forEach` functor, you can use parameter destructuring to shrink code even more.